ttnn/ttnn/operations/conv1d.py

```diff
@@ -30,6 +30,8 @@ def Conv1d(
     conv_config: Conv1dConfig = None,  # config overrides by user
     conv_op_cache={},  # basic conv object caching in python needed for intermediate refactoring. Not needed after full op refactoring in C++.
     debug=False,
+    return_output_length=False,
+    return_prepared_device_weights=False,
```

Comment:
Why complicate the API like this?

If the values are computed then there's no additional cost to returning them.

Same comment below for 2d.